README.md

```diff
@@ -417,13 +417,13 @@ sudo -E -u www-data php -dxdebug.remote_host=192.168.21.1 occ
 
 ### Useful commands
 
-- Restart apache to reload php configuration without a full container restart: `docker-compose kill -s USR1 nextcloud`
-- Access to mysql console: `mysql -h $(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_database-mysql_1) -P 3306 -u nextcloud -pnextcloud`
+- Restart Apache to reload php configuration without a full container restart: `docker-compose kill -s USR1 nextcloud`
+- Access to MySQL console: `mysql -h $(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_database-mysql_1) -P 3306 -u nextcloud -pnextcloud`
 - Run an LDAP search: `ldapsearch -x -H ldap://$(docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' nextcloud_ldap_1) -D "cn=admin,dc=planetexpress,dc=com" -w admin -b "dc=planetexpress,dc=com" -s subtree <filter> <attrs>`
 
-## Keycloak
+## [Keycloak}(https://www.keycloak.org/)
 
-- Keycloak is using ldap as a user backend (make sure the ldap container is also running)
+- Keycloak is using LDAP as a user backend (make sure the LDAP container is also running)
 - `occ user_oidc:provider Keycloak -c nextcloud -s 09e3c268-d8bc-42f1-b7c6-74d307ef5fde -d https://keycloak.local.dev.bitgrid.net/auth/realms/Example/.well-known/openid-configuration`
 - https://keycloak.local.dev.bitgrid.net/auth/realms/Example/.well-known/openid-configuration
 - nextcloud
```

Comment:
Not exactly sure what is intended with these three last lines...

The URL also seems problematic currently, with a Service Unavailable error.

Reply:
This is still a left over from my initial personal setup, so those should actually be using the default domain suffix so would be `http://keycloak.dev.local/...`

Reply:
I say lets merge this an I'll create another PR w the Keycloak URL changes - esp since they should require some testing before being merged..

That allows this relatively easy PR to merge easily, and the more involved to be merged as appropriate.

🚀